pkg/dd/interface.go

```diff
@@ -0,0 +1,10 @@
+package dd
+
+import "time"
+
+type DatadogWorkflowInterface interface {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type DatadogWorkflowInterface should have comment or be unexported](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)